Cargo.toml

```diff
@@ -41,8 +41,10 @@ rpath = false
 blake2 = { git = "https://github.com/near/near-blake2.git", version = "0.9.1", default-features = false }
 borsh = { version = "0.8.2", default-features = false }
 bn = { package = "aurora-bn", git = "https://github.com/aurora-is-near/aurora-bn.git", default-features = false }
-evm = { version = "0.26.0", default-features = false }
-evm-core = { version = "0.26.1", default-features = false }
+evm = { path = "../evm", default-features = false }
+#evm = { version = "0.26.0", default-features = false }
+evm-core = { path = "../evm/core", default-features = false }
+#evm-core = { version = "0.26.1", default-features = false }
```

Comment:
I will point to the right dependency when everything is settled. For now this is depending on this changes: https://github.com/aurora-is-near/sputnikvm/pull/2

Reply:
It is pointing right now, to fixed github commit.

Reply:
The change is already upstream, so poiting to evm 0.28.0 now